contracts/DCAStrategies/DCAStrategies/DCAStrategiesPositionsHandler.sol

```diff
@@ -9,6 +9,11 @@ abstract contract DCAStrategiesPositionsHandler is IDCAStrategiesPositionsHandle
 
   mapping(uint256 => Position) internal _userPositions;
 
+  modifier onlyWithPermission(uint256 _positionId, IDCAStrategies.Permission _permission) {
```

Comment:
Nit: since this is only for internal purposes, could you move it to the bottom of the file?